src/newSeed/baseStage.scss

```diff
@@ -539,7 +539,12 @@
             background-color: $Shade03;
           }
         }
+
+        + * + * {
```

Comment:
what does this selector do?

Reply:
Same as `:not(:first-child)` but shorter...

Reply:
shorter is not always better.  can we please use the long version since it is very clear in meaning?

Reply:
done.